libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc

```diff
@@ -7,7 +7,7 @@
 
 The `add_kubernetes_metadata` processor annotates each event with relevant
 metadata based on which Kubernetes pod the event originated from.
-At startup it detects an `in_cluster` environment and caches the
+At startup, it detects an `in_cluster` environment and caches the
```

Comment:
A comma is OK, but the general rule is to add a comma after an introductory phrase of 4 or more words.